### ISSUE TITLE
Models

### ISSUE THREAD

**Issue by joe-sht** [google-research/magvit#15]
Hi, all! I met an issue on running
python main.py --workdir=../out --config=configs/maskgvt_bair_config.py
My goal is training some model on bair test set for frame prediction.
My issue is like no access to gs://magvit/models/bair_3d_base
I checked that this model is needed on evaluation step and tried to switch off evaluation.
But I still have issues and script looks not run without these models.
As I understand these models have not released yet. But is it possible to train some model without these ones
based only on downloaded 30 Gb Bair dataset?

**Comment by roadjiang** [comment]
Hi you should be able to train the BAIR model from scratch using a couple of GPUs.

As for the model pretrained weight release gs://magvit/models/bair_3d_base
Please see this [note](https://github.com/google-research/magvit/issues/16)

**Comment by shinshiner** [comment]
> Hi you should be able to train the BAIR model from scratch using a couple of GPUs.
> 
> As for the model pretrained weight release gs://magvit/models/bair_3d_base Please see this [note](https://github.com/google-research/magvit/issues/16)

how long it takes to train the base/large model according to the configurations present in supplement files?
